### PR TITLE
docs: update readme

### LINKED ISSUE #98: Docs need updating with how to use APPROVED_TOKENS
To authenticate with the API, an envar needs to be passed

```
APPROVED_TOKENS
```

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #98

### CODE REVIEW

**Review by nhcarrigan — APPROVED** [review]
I don't think I have write access on this one.

File: README.md [hunk 1]
```diff
@@ -83,6 +83,14 @@ If this variables is not true or undefiend no Logs will be made.
 
 When set to `true` the App will Log every incoming request.
 
+## Authentication
+
+Authentication is handled via an environment-variable called `APPROVED_TOKENS`.
+The approved tokens which are validated by the API are put there.
+In case there should be multiple tokens, for multiple clients, they need to be **comma-seperated**.
+
+To use Authetication the token needs to be passed in the `token`-header of the client-request.
```
Comment (nhcarrigan) [inline README.md:92]:
```suggestion
To use Authentication the token needs to be passed in the `token`-header of the client-request.
```
Typo.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — APPROVED** [review]
Another LGTM for what it's worth :)

**Comment by nhcarrigan** [comment]
image

Hmm is this a flaky test? 🤔

**Comment by Cahllagerfeld** [comment]
I honestly dont understand why the checks fail. I didnt even change anything at the coding :o

**Comment by nhcarrigan** [comment]
Yeah, an `ECONNRESET` error definitely suggests flaky, too. Do you have write access? If so, you should be able to re-run those checks.

**Comment by Cahllagerfeld** [comment]
On the main-branch everything passed. What could the reason for this be?

**Comment by nhcarrigan** [comment]
Yep the one you re-ran passed this time. It's just a flaky test.

**Comment by Cahllagerfeld** [comment]
I'm confused xD

**Comment by nhcarrigan** [comment]
A test that randomly fails because it doesn't like us. 🤣

**Comment by nhcarrigan** [comment]
You want to re-run [this one](https://github.com/EddieHubCommunity/api/pull/99/checks?check_run_id=2581258864) too?

**Comment by Cahllagerfeld** [comment]
> 
> 
> A test that randomly fails because it doesn't like us. 🤣

There seems to be a second one, doesnt it? 
image

**Comment by nhcarrigan** [comment]
Yeah because you're working off a branch in this repo, we get the actions running on `push` and on `pull_request`. Duplicate testing :D

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Comment by eddiejaoude** [comment]
Thanks all 👍

**Pull request merged**